dbms/include/DB/Interpreters/DDLWorker.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include <DB/Interpreters/Context.h>
+#include <common/logger_useful.h>
+
+#include <atomic>
+#include <chrono>
+#include <condition_variable>
+#include <mutex>
+#include <thread>
+
+namespace DB
+{
+
+class DDLWorker
```

Comment:
Need comment about what it is.
What is the structure of nodes in ZooKeeper. How they are processed.
And so on.